content/blog/2021/05/new-camel-k-logging-features/index.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "New Camel K Logging Features"
+date: 2021-05-10
+authors: ["orpiske"]
+categories: ["Features", "Camel K"]
+preview: "New Camel K Logging Features"
+summary: "New Camel K Logging Features for Camel K 1.5"
+---
+
+
+As part of our continuous work to improve Camel K and improve its stability, features and
+usefulness to our community, we recently worked on logging improvements for different
+parts of Camel K and the integrations it generated.
+
+During the 1.5 development release we dedicated some time to review the logging capabilities
+of Camel K. This version introduces a new [logging trait](https://camel.apache.org/camel-k/latest/traits/logging.html) that simplifies access to the logging configuration available on the runtime.
+
+## Colorized logs
+
+By default, Camel K will colorize the log output, making it easier to distinguish between
+log levels, loggers, timestamps and the logging message. Naturally, we understand that not
+everyone likes colorized logs. As such it's possible to turn it by using the logging.color
+trait option.
+
+This is what it looks like:
+
+![Colorized Camel K Logs](colorized-camel-k-logs.png)
+
+
+## Easier support for additional debug levels
+
+Additional log levels can be turned on or off at ease, using the logging.level trait. Users
+can inform the minimum log level (i.e.: `TRACE`, `DEBUG`, `INFO`, etc) and control how much
+information is presented by the logger.
+
+
+## Custom logging format
+
+Leveraging the previous features, it's also possible to control the format of the message
+displayed. Since the runtime leverages Camel Quarkus and, by extension, Quarkus itself, all
+the logging format options are available to customize the log messages.
```

Comment:
So here, we are talking about -t logging.format ?

Reply:
I am trying to explain that the logging format options are available (i.e.: `%d{xxx} %H %m`, etc that we can use to customize the logging fields.

Reply:
I think I'll rephrase this...

Reply:
I like the idea about adding an example. I'll include that!